### ISSUE TITLE
Classification of Arachnida

### ISSUE THREAD

**Issue by aoern** [CatalogueOfLife/data#187]
@gdower 
The classification of Arachnida in DwC Sep-1 edition is a mess clearly seen in the legacy UI's taxonomic tree:

- Phylum Arthropoda does not show class Arachnida as its child. Instead, the arachnid taxa are classified under placeholder class "Not assigned".

- Orders Holothyrida, Mesostigmata, Opiliones, Sarcoptiformes and Trombidiformes are shown as immediate children of phylum Arthropoda with no species.

- Orders Holothyrida and Opiliones under class "Not assigned" have no species.

**Comment by mdoering** [comment]
As you can see in https://data.catalogue.life/dataset/3/classification?taxonKey=7a77dbd5-bcff-42c9-b9fc-b05d5678302f--incertae-sedis--SUBCLASS
Arachnida is treated as a subclass now, I assume due ti ITIS:
https://itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=1117840#null

It is weird though that it does not have a class assigned. @yroskov should this not be Euchelicerata then?

**Comment by yroskov** [comment]
Indeed, Arachnida is a subclass according ITIS. Old database structure, which adopted only taxa of basic ranks, is still in use for a final product. It eliminates subclass and places all children taxa in Not Assigned node. 

Unfortunately, mess in CoL+ classification will grow. I have warned Global Team when allowance of intermediate ranks was discussed.

**Comment by aoern** [comment]
@yroskov But what explains the empty orders?

**Comment by yroskov** [comment]
Hmm, you are right @aoern. 

Two different issues: 
1) Order Holothyrida have no species in ITIS. It is ITIS global checklist in progress.

2) Sector Opiliones was not established as shown in assembly Tree now. It might be my mistake, or something happened in the Clearinghouse independently from me. At least, all new ITIS classification down to genus level is present in the Tree. So, sector was established and data were synced once, then sector disappeared (@gdower @mdoering).

Now I re-established sector Opiliones and synced data. Opiliones checklist is equal to ITIS now: some families & genera have species, others - not. It is another ITIS checklist in progress.

**Comment by yroskov** [comment]
Update: I have double checked ITIS list of global checklists, there is no order Opiliones in it. Data were correct: no Opiliones as a sector. I have killed it now. Only ITIS classification will be present there; no species.

**Comment by aoern** [comment]
Ok. In 2019 annual order Opiliones had 1280 species out of estimated 6571, from ITIS Global and ITIS Regional. The coverage was only 19 %. And now CoL has lost all of these familiar long-legged "harvestman" fellows!

**Comment by mdoering** [comment]
So Opiliones has been dropped as it was regional only, correct? Should the order down to genus then be present in COL again, it is entirely missing now. There are 1280 species in ITIS:
https://data.catalogue.life/dataset/2144/taxon/82753

@yroskov I suggest you create a new sector then and only sync ranks down to genus level.
With this you can easily keep up to date to the latest ITIS classification without having to create & delete sectors just to remove the species.

As for estimations this paper from 2013 gives a global number of 6534 species:
https://www.biotaxa.org/Zootaxa/article/view/zootaxa.3703.1.7

**Comment by mdoering** [comment]
There are global lists of suborders existing, that ITIS does not cover in its entirety.

Suborder CYPHOPHTHALMI with 113 species ([ITIS has only 14](https://data.catalogue.life/dataset/2144/taxon/1012396)):
http://sea-entomologia.org/PDF/RIA_2/R02-002-049.pdf

And for suborder Dyspnoi the informative [Schönhofer paper from 2013](https://www.mapress.com/zootaxa/2013/f/zt03679p068.pdf) gives 286 species ([ITIS lists 37](https://data.catalogue.life/dataset/2144/taxon/1012397)).

It also mentions Joel Hallan (2005) which I could not find online though in my quick initial search:

> When searching for taxonomic-systematic online resources on Opiliones one quickly discovers an extensive list managed by Joel Hallan (2005; the 'Hallan list') providing taxonomic and systematic information on many groups of animals. Its contribution to Opiliones is comprehensive, and the Hallan list is presently the only available digital entity, which gives most taxon names together with relevant taxonomic and additional information. Even with errors introduced, additional taxa described and systematics partly changed, these issues are recognisable for taxonomic experts, to whom the Hallan list serves as a good template.


Kury did a synopsis of ~100 catalogs and checklists of Opiliones:
https://www.biotaxa.org/Zootaxa/article/view/zootaxa.3184.1.2

**Comment by mdoering** [comment]
I agree with @chantalhuijbers we need to reopen this issue as there are 2 major issues still unresolved:

- [ ] subclass Arachnida has no class. It makes no sense to treat it as a subclass in that case.
- [ ] no species for Opiliones. We have 374 genera, but no species. This is a major loss for an import, globally omnipresent group. I have provided links above to fix at least the major suborders. I have created a new issue #220 for this

This leaves the missing class issue as the only one open here.

**Comment by DaveNicolson** [comment]
While it is true that nobody has claimed the whole of Opiniones as a GSD sector, ITIS has done a fair amount of work to start on this, and four of the superfamilies (with nearly 1000 valid species) were completed in ITIS some time ago, as I tried to note in the Google Sheet with the new classification for Arachnida, a portion of which I will note here... for some reason it won't let me paste an image here or upload one either, so I will try with a data paste... BOLDING shows the completed superfamilies:

Order: Opiliones [82753] [parts are GSD] (48-50 families) |   |   |  
-- | -- | -- | --
  | Suborder: Cyphophthalmi [1012396] (6 families) |   |  
  |   | Infraorder: Boreophthalmi [1012399] |  
  |   | Infraorder: Scopulophthalmi [1012402] |  
  |   | Infraorder: Sternophthalmi [1012403] |  
  | Suborder: Dyspnoi [1012397] (7-8 families) |   |  
  |   |   | Superfamily: Acropsopilionoidea [1012408]
  |   |   | Superfamily: Ischyropsalidoidea [1012413]
  |   |   | Superfamily: Troguloidea [1012416]
  | Suborder: Eupnoi [1012398] (4-5 families) |   |  
  |   |   | Superfamily: Caddoidea [1012410]
  |   |   | Superfamily: Phalangioidea [1012414]
  | Suborder: Laniatores [1012395] [parts are GSD] (31 families) |   |  
  **|   | Infraorder: Insidiatores (5 families) [no infraorder in ITIS, but superfams are there] |  
  |   |   | Superfamily: Travunioidea [1012404] (3 families)
  |   |   | Superfamily: Triaenonychoidea [1012405] (2 families)**
  |   | Infraorder: Grassatores [1012400] [parts are GSD] (26 families) |  
  |   |   | Superfamily: Assamioidea [1012409] (2 families)
  |   |   | Superfamily: Epedanoidea [1012411] (5 families)
  |   |   | Superfamily: Gonyleptoidea [1012412] (9 families)
  |   |   | Superfamily: Phalangodoidea [1012415] (1 family)
  **|   |   | Superfamily: Samooidea [1012406] (5 families)
  |   |   | Superfamily: Zalmoxoidea [1012407] (4 families)**

The completed superfamilies are:
Superfamily: Travunioidea [1012404] (3 families)
Superfamily: Triaenonychoidea [1012405] (2 families)
Superfamily: Samooidea [1012406] (5 families)
Superfamily: Zalmoxoidea [1012407] (4 families)

I suppose we could call it an "aspirational" GSD for the whole group (Opiliones), but I don't care for that approach here, since at _present_ ITIS is not focusing on Opiliones, but I hope to do so at some point.

Not sure how much this helps, but hopefully some...

Thanks,
Dave

**Comment by DaveNicolson** [comment]
And note that the new classification for Infraclass Arachnida DOES include a class, it should be under Class Euchelicerata.
Phylum Arthropoda
_Subphylum Chelicerata
__Class Euchelicerata
___Subclass Arachnida...

**Comment by DaveNicolson** [comment]
(re Opiliones classification, note that Infraorder Insidiatores has been dropped from the classification based on Kury 2014, who stated, "For decades, Travunioidea and Triaenonychoidea have been historically regarded as a single unit, called Travunioidea (e.g., Martens 1980). Kury (2003) resurrected the name Insidiatores Loman 1900 to collectively refer to them, but subsequent authors failed to recover the monophyly of Insidiatores (Giribet and Kury 2007) and even of the component families (Sharma and Giribet 2011)." That is why we don't use it in ITIS at present. Hopefully things will continue to resolve over time, but that's where we were in our last update including those superfamilies)
